judge/models/contest.py

```diff
@@ -70,13 +70,15 @@ class Contest(models.Model):
                                                  'specified organizations.'))
     is_rated = models.BooleanField(verbose_name=_('contest rated'), help_text=_('Whether this contest can be rated.'),
                                    default=False)
-    hide_scoreboard = models.BooleanField(verbose_name=_('hide scoreboard'),
-                                          help_text=_('Whether the scoreboard should remain hidden for the duration '
-                                                      'of the contest.'),
-                                          default=False)
     view_contest_scoreboard = models.ManyToManyField(Profile, verbose_name=_('view contest scoreboard'), blank=True,
                                                      related_name='view_contest_scoreboard',
                                                      help_text=_('These users will be able to view the scoreboard.'))
+    scoreboard_visibility = models.CharField(verbose_name=_('scoreboard visibility'), default='Visible', max_length=32,
+                                             help_text=_("Scoreboard visibility through the duration of the contest"),
+                                             choices=[
+                                                 ('Visible', _('Visible')),
```

Comment:
There is no need to make the actual string in the database more than one or two characters long. They should be introduced as constants on the class to avoid typos.